files/faban/src/cloud/benchflow/collectors/faban.go

```diff
@@ -51,8 +51,8 @@ func signalOnKafka(minioKey string, containerID string) {
 	}
  
 func backupHandler(w http.ResponseWriter, r *http.Request) {
-    query := r.FormValue("path")
-    path := query
+    fabanPath := r.FormValue("path")
+    path := "/app/faban/output/"+fabanPath
```

Comment:
Move this definition at file level, and add a trailing slash `/` because it is not passed in the query.